### PR TITLE
fix(html): Fix various invalid HTML

### PR DESCRIPTION
**Pull request by christopherthielen**
Found and fixed some more invalid HTML

see #6597

### CODE REVIEW

**Comment by christopherthielen** [comment]
@gcomstock since you seem to care about this sort of thing :)

File: app/scripts/modules/ecs/src/instance/details/instanceDetails.html [hunk 1]
```diff
@@ -137,13 +139,6 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
         </dd>
       </dl>
     </collapsible-section>
-    <collapsible-section heading="Console Output" ng-if="baseIpAddress">
-      <ul>
-        <li>
-          <console-output-link instance="instance"></console-output-link>
-        </li>
-      </ul>
-    </collapsible-section>
```
Comment (gcomstock) [inline app/scripts/modules/ecs/src/instance/details/instanceDetails.html]: Why is this getting removed?

File: app/scripts/modules/ecs/src/serverGroup/events/events.html [hunk 2]
```diff
@@ -11,21 +11,19 @@ <h3>Server Group Events for {{$ctrl.serverGroup.name}}</h3>
...
     <div ng-if="!$ctrl.viewState.loading && !$ctrl.viewState.error && !$ctrl.events.length" class="text-center">
-      No scaling events found for {{$ctrl.serverGroup.name}}.</p>
+      <p>No scaling events found for {{$ctrl.serverGroup.name}}.</p>
     </div>
 
     <div ng-repeat="event in $ctrl.events" ng-if="!$ctrl.viewState.loading">
-        <p class="clearfix">
-          <div ng-if="event.status ==git ad= 'Success'">
+      <p class="clearfix"></p>
+          <div ng-if="event.status === 'Success'">
```
Comment (gcomstock) [inline app/scripts/modules/ecs/src/serverGroup/events/events.html]: I dont know how these angular conditionals work but this change looks suspicious

File: app/scripts/modules/ecs/src/instance/details/instanceDetails.html (hunk 1 above)
Comment (christopherthielen) [reply app/scripts/modules/ecs/src/instance/details/instanceDetails.html]: good question, didn't mean to. 
glad you reviewed this :)

File: app/scripts/modules/google/src/loadBalancer/configure/http/healthCheck/healthChecks.html [hunk 3]
```diff
@@ -3,10 +3,10 @@
     <div class="alert alert-warning">
       <p><i class="fa fa-exclamation-triangle"></i>
         The following health checks have not been associated with a backend service:
-        <ul>
-          <li ng-repeat="service in ctrl.command.getUnusedHealthChecks(ctrl.command)" ng-bind-html="service"></li>
-        </ul>
       </p>
+      <ul>
+        <li ng-repeat="service in ctrl.command.getUnusedHealthChecks(ctrl.command)" ng-bind-html="service"></li>
+      </ul>
```
Comment (gcomstock) [inline app/scripts/modules/google/src/loadBalancer/configure/http/healthCheck/healthChecks.html]: This p tag might be applying styles to the ul that we're losing here, worth a look

File: app/scripts/modules/ecs/src/serverGroup/events/events.html (hunk 2 above)
Comment (christopherthielen) [reply app/scripts/modules/ecs/src/serverGroup/events/events.html]: The old markup is nonsense and I suspect a typo:  
```
 <div ng-if="event.status ==git ad= 'Success'">
```

File: app/scripts/modules/openstack/src/serverGroup/details/serverGroupDetails.html [hunk 4]
```diff
@@ -166,7 +166,6 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
         <dd ng-repeat-end>{{value}}</dd>
       </dl>
     </collapsible-section>
-  </div>
```
Comment (gcomstock) [inline app/scripts/modules/openstack/src/serverGroup/details/serverGroupDetails.html]: suspicious

File: app/scripts/modules/ecs/src/serverGroup/events/events.html (hunk 2 above)
Comment (gcomstock) [reply app/scripts/modules/ecs/src/serverGroup/events/events.html]: right on

File: app/scripts/modules/google/src/loadBalancer/configure/http/healthCheck/healthChecks.html (hunk 3 above)
Comment (christopherthielen) [reply app/scripts/modules/google/src/loadBalancer/configure/http/healthCheck/healthChecks.html]: I'm not worried about it.  `<ul>` is not legal inside a `<p>` tag (only inline content is legal).  

When the browser parses this, it puts the `<ul>` as a peer to (not inside of) the  `<p>` tag.

screen shot 2019-02-21 at 4 50 41 pm

**Comment by gcomstock** [comment]
Look at that! Thanks for the tip.

File: app/scripts/modules/google/src/loadBalancer/configure/http/healthCheck/healthChecks.html (hunk 3 above)
Comment (gcomstock) [reply app/scripts/modules/google/src/loadBalancer/configure/http/healthCheck/healthChecks.html]: Look at that! Thanks for the tip.

File: app/scripts/modules/openstack/src/serverGroup/details/serverGroupDetails.html (hunk 4 above)
Comment (christopherthielen) [reply app/scripts/modules/openstack/src/serverGroup/details/serverGroupDetails.html]: suspicious? What do you mean?

Comment (gcomstock) [reply app/scripts/modules/openstack/src/serverGroup/details/serverGroupDetails.html]: Just wanted to make sure prettier wasn't removing a closing tag erroneously and possibly breaking the layout. It wasn't obvious to me if this was just an extraneous tag or not.

Comment (christopherthielen) [reply app/scripts/modules/openstack/src/serverGroup/details/serverGroupDetails.html]: ah I see.  This commit wasn't prettier, this was all by hand to fix the HTML spec issues prettier pointed out

Comment (gcomstock) [reply app/scripts/modules/openstack/src/serverGroup/details/serverGroupDetails.html]: Ah! Sorry to waste yer time

**Pull request merged**